packages/analytics-browser/src/plugins/network-connectivity-checker.ts

```diff
@@ -33,6 +33,14 @@ export const networkConnectivityCheckerPlugin = (): BeforePlugin => {
   };
 
   const setup = async (config: BrowserConfig, amplitude: BrowserClient) => {
+    if (typeof navigator === 'undefined') {
+      config.loggerProvider.debug(
+        'Network connectivity checker plugin is disabled because navigator is not available.',
+      );
+      config.offline = false;
```

Comment:
As per your comment in my PR I think you wanted this to be `true`?

Reply:
The comment was to make it false by default. We want to be online if the network connectivity checker plugin was not available.

https://github.com/amplitude/Amplitude-TypeScript/pull/721/files#r1603984757